### PR TITLE
doc / probit korea

### PR DESCRIPTION
**Pull request by JeremyKono**
added probit korea:
image

### CODE REVIEW

**Comment by CrimsonJacket** [comment]
The prompt for connecting to ProBit KR is as seen below. I also noticed that it is `Client ID >>>` for instead of `API Key>>>` for both ProBit Global and ProBit Korea.

Also can you change ProBit in the toc and the headers to **ProBit Global**

image

File: content/exchange-connectors/probit-korea.mdx [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: ProBit Korea (BETA)
+description: About ProBit connector
+---
+
+import Callout from "../../src/components/Callout";
+
+ProBit Exchange is a Top 20 crypto exchange globally. We have completed over 200 rounds of IEO and have been consistently ranked Top 4 in Korea. ProBit Exchange provides unlimited trading access highlighted by nearly 1,000 trading pairs.
```
Comment (CrimsonJacket) [inline content/exchange-connectors/probit-korea.mdx:8]: Change `ProBit` to `ProBit KR`.

File: content/exchange-connectors/probit-korea.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+Enter your ProBit secret key >>>
+```
+
+Private keys and API keys are stored locally for the operation of the Hummingbot client only. At no point will private or API keys be shared to CoinAlpha or be used in any way other than to authorize transactions required for the operation of Hummingbot.
+
+<Callout
+  type="tip"
+  body="For copying and pasting into Hummingbot, see [this page] for more instructions in our Hummingbot Help Center."
+  link={[
+    "https://hummingbot.zendesk.com/hc/en-us/articles/900004871203-Copy-and-paste-your-API-keys",
```
Comment (CrimsonJacket) [inline content/exchange-connectors/probit-korea.mdx:43]: Update this link to
https://support.probit.kr/hc/ko/articles/900003084746-2-%EB%8B%A8%EA%B3%84-%EC%9D%B8%EC%A6%9D-2FA-Google-OTP-%EB%A5%BC-%EC%82%AC%EC%9A%A9%ED%95%98%EB%8A%94-%EB%B0%A9%EB%B2%95-

File: content/exchange-connectors/probit-korea.mdx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  body="This connector has undergone code review, internal testing and was shipped during one of our most recent releases. As part of User Acceptance Testing, we encourage user to report any issues with this connector to our [Discord server] or [submit a bug report]"
+  link={[
+    "https://discord.com/invite/2MN3UWg",
+    "https://github.com/CoinAlpha/hummingbot/issues/new?assignees=&labels=bug&template=bug_report.md&title=",
+  ]}
+/>
+
+## Using the Connector
+
+To use [ProBit](https://www.probit.kr/login?) connector, you will need to generate and provide your API key in order to trade using Hummingbot.
```
Comment (CrimsonJacket) [inline content/exchange-connectors/probit-korea.mdx:30]: `Probit` -> `ProBit KR` and change the link to https://www.probit.kr

File: content/exchange-connectors/probit-korea.mdx [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  type="tip"
+  body="For copying and pasting into Hummingbot, see [this page] for more instructions in our Hummingbot Help Center."
+  link={[
+    "https://hummingbot.zendesk.com/hc/en-us/articles/900004871203-Copy-and-paste-your-API-keys",
+  ]}
+/>
+
+### Creating ProBit API Keys
+
+1. Log in to your account [here](https://www.probit.com/login?locale=en-us&url=/exclusive), then click your My Page > API Management (If you do not have an account, you will have to create one and enable 2FA.)
```
Comment (CrimsonJacket) [inline content/exchange-connectors/probit-korea.mdx:49]: Change link to `https://www.probit.kr/login?`

**Review by CrimsonJacket — CHANGES_REQUESTED** [review]
Some minor changes requested to links and certain content headers.

**Review by CrimsonJacket — CHANGES_REQUESTED** [review]
You're still missing `KR` before `Client ID`
image
 ie. `Enter your Probit KR Client ID >>>`

**Comment by JeremyKono** [comment]
Everything has been resloved in the recent commits.

**Pull request merged**